modules/apps/captcha/captcha-taglib/src/main/java/com/liferay/captcha/taglib/servlet/taglib/CaptchaTag.java

```diff
@@ -76,6 +81,23 @@ private String _getURL(HttpServletRequest httpServletRequest) {
 
 		String url = themeDisplay.getPathMain() + "/portal/captcha/get_image";
 
+		try {
+			CaptchaConfiguration captchaConfiguration =
+				ConfigurationProviderUtil.getSystemConfiguration(
+					CaptchaConfiguration.class);
+
+			if ((captchaConfiguration != null) &&
```

Comment:
no need to check null because the exception will be throws

Reply:
Fixed it. Thanks